.github/workflows/auto-context.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         fi
 
     - name: Create Pull Request
-      if: steps.update.outputs.create_pull_request == 'true'
+      if: {{ steps.update.outputs.create_pull_request == 'true' }}
```

Comment:
When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. For more information about if conditionals, see [Workflow syntax for GitHub Actions.](https://docs.github.com/en/free-pro-team@latest/articles/workflow-syntax-for-github-actions/#jobsjob_idif)

Reply:
updated